src/pages/index.vue

```diff
@@ -6,20 +6,29 @@
     <header
       class="flex-grow w-full lg:w-auto lg:min-w-[32rem] xl:min-w-[64rem] box-border flex flex-col justify-between lg:justify-center"
     >
-      <VLogoButton class="lg:hidden ms-3" :auto-resize-logo="false" />
+      <VLogoButton
```

Comment:
Neither header nor `VLogoButton` have a color class, so the color of the logo becomes black instead of dark charcoal.